com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -211,6 +211,8 @@ void SetPerCameraShaderVariables(CommandBuffer cmd, ref CameraData cameraData)
             cmd.SetGlobalVector(ShaderPropertyId.scaledScreenParams, new Vector4(scaledCameraWidth, scaledCameraHeight, 1.0f + 1.0f / scaledCameraWidth, 1.0f + 1.0f / scaledCameraHeight));
             cmd.SetGlobalVector(ShaderPropertyId.zBufferParams, zBufferParams);
             cmd.SetGlobalVector(ShaderPropertyId.orthoParams, orthoParams);
+
+            cmd.SetGlobalVector(ShaderPropertyId.screenSize, new Vector4(cameraWidth, cameraHeight, 1.0f / cameraWidth, 1.0f / cameraHeight));
```

Comment:
Do we need a new property? Can't we reuse `screenParams` a few lines up?

Reply:
We could, but sadly zw values a bit different that has +1. This is artifact from old pipeline where +1 was needed for some calculations. I want to avoid the cost in shader subtracting that +1.
Deferred was already using this property too, so I moved it to our core as this is very useful property.

Reply:
I think there's been talk about removing that +1 before. Maybe this could be changed in a later PR.